frontend/tests/controllers/ContestUpdateTest.php

```diff
@@ -1928,4 +1928,93 @@ public function testUpdatePenaltyTypeFromAContestWhenProblemVersionIsUpdated(
         }
         $this->assertSame($runs[0]['penalty'], $expectedPenalty);
     }
+
+    /**
+     * A PHPUnit data provider for testing contest recommended flag permissions.
+     *
+     * @return list<array{0: string, 1: bool}>
+     */
+    public function contestRecommendedPermissionsProvider(): array {
+        return [
+            ['support', true],
```

Comment:
Is it possible to use the group aliases here?
```suggestion
            [\OmegaUp\Authorization::SUPPORT_GROUP_ALIAS, true],
            ...
```
This way, we can avoid an extra step

Reply:
Right 🫡